psychrnn/backend/curriculum.py

```diff
@@ -145,11 +145,20 @@ def metric_test(self, input_data, correct_output, output_mask, test_output, epoc
 			return True
 		return False
 
-	def get_generator_function(self):
-		""" Return the generator function for the current task.
+	def batch_generator(self):
```

Comment:
The problem is that the name `get_generator_function` is misleading. It returns a generator iterator rather than a generator function. We may keep the old name and only change the doc string. Or we can have the old method call the new `batch_generator` and print a deprecation warning.

Reply:
Having the old method call the new batch_generator with a deprecation warning seems like a good option.